src/components/Button/Button.jsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import './Button.scss';
+
+const Button = ({
+  id,
+  className,
+  children,
+  isDisabled,
+  fullWidth,
+  onClick,
+  tabIndex,
+  isLoading,
+  onFocus,
+  onBlur,
+}) => {
+  const inputId = id || uuid();
+
+  const disabled = isLoading || isDisabled;
+
+  const buttonClasses = classNames('Palmetto-Button', className, {
+    'loading': isLoading,
```

Comment:
you don't need the quotes on these unless you are interpolating a variable, or using dashes classnames will read the object key.